lib/licenses.nix

```diff
@@ -877,6 +877,28 @@ in mkLicense lset) ({
     fullName = "Non-Profit Open Software License 3.0";
   };
 
+  nvidiaCuda = {
+    shortName = "CUDA EULA";
+    fullName = "CUDA Toolkit End User License Agreement (EULA)";
+    url = "https://docs.nvidia.com/cuda/eula/index.html#cuda-toolkit-supplement-license-agreement";
+    free = false;
+  };
+
+  nvidiaCudaRedist = {
+    shortName = "CUDA EULA";
```

Comment:
```suggestion
    shortName = "CUDA EULA Redistributable";
```

Reply:
Do we only use `shortName` ourselves to distinguish between licenses? Because NVidia doesn't explicitly call it that